docs/Shadows.mdx

```diff
@@ -45,7 +45,7 @@ As well as the neutral colors for shadow 2 other colors exist that are used for
 Using both size and color tokens, different shadows can be applied to components.
 
 <Canvas>
-  <Story id="shadows-shadows--usage" />
+  <Story id="shadows-shadows--example-usage" />
 </Canvas>
 
 | Component                | JS                                                                              | CSS                                                              |
```

Comment:
Fixing broken story in MDX docs

### Before

<img width="1440" alt="Screen Shot 2022-09-06 at 12 27 45 PM" src="https://user-images.githubusercontent.com/8112138/188722122-d96b2f27-c1f3-4658-bab3-4b16f5156368.png">


### After

<img width="1440" alt="Screen Shot 2022-09-06 at 12 27 11 PM" src="https://user-images.githubusercontent.com/8112138/188722139-3c56e2cd-eced-4ee3-8f4a-329aea4935af.png">